chopsticks/configs/acala.yml

```diff
@@ -0,0 +1,103 @@
+endpoint: wss://acala-rpc.aca-api.network
+mock-signature-host: true
+runtime-log-level: 5
+block: 7000000
+wasm-override: ./chopsticks/wasm/acala-2250.wasm
```

Comment:
```suggestion
wasm-override: ./chopsticks/wasm/acala-2260.wasm
```

Reply:
old tests worked with runtime 2250, so let's control variable and upgrade to 2260 **after** all tests are converted and running (otherwise if some tests fail, we don't know if it's the test's issue, or runtime 2260's issue).